src/libs/OptionsListUtils.js

```diff
@@ -216,18 +216,19 @@ function isPersonalDetailsReady(personalDetails) {
 function getParticipantsOptions(report, personalDetails) {
```

Comment:
I'm curious why we're not returning `isMoneyRequestReport: true` here, but that's for another day

Reply:
Asked [here](https://github.com/Expensify/App/pull/17132/files#r1235347684), but it's been a few months so idk if there's a good answer, just checking